tests/foreman/longrun/test_oscap.py

```diff
@@ -42,39 +42,15 @@
 from robottelo.constants import OSCAP_PERIOD
 from robottelo.constants import OSCAP_PROFILE
 from robottelo.constants import OSCAP_WEEKDAY
-from robottelo.decorators import skip_if_not_set
 from robottelo.helpers import add_remote_execution_ssh_key
 from robottelo.helpers import file_downloader
 from robottelo.helpers import ProxyError
-from robottelo.test import CLITestCase
 from robottelo.vm import VirtualMachine
 
 
-class OpenScapTestCase(CLITestCase):
+class TestOpenScap:
```

Comment:
if there is just one class in this file, is the class truly necessary?

Reply:
updated the pr.